admin/app/controllers/authenticated/trainings/new.js

```diff
@@ -0,0 +1,34 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Controller from '@ember/controller';
+
+export default class NewController extends Controller {
+  @service notifications;
+  @service store;
+  @service router;
+
+  @action
+  goBackToTrainingList() {
+    this.router.transitionTo('authenticated.trainings.list');
+  }
+
+  @action
+  async createOrUpdateTraining(trainingDTO) {
+    const training = this.model;
+    training.title = trainingDTO.title;
+    training.link = trainingDTO.link;
+    training.type = trainingDTO.type;
+    training.duration = trainingDTO.duration;
+    training.locale = trainingDTO.locale;
+    training.editorLogoUrl = trainingDTO.editorLogoUrl;
+    training.editorName = trainingDTO.editorName;
+
+    try {
+      await training.save();
```

Comment:
C'est pas plus intéressant de créer le record ici et pas dans le modèle ? 
`await this.store.createRecord('training', ...training).save();`